mautic/mautic
# Add back support for importing entity properties

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

When importing a contact or company CSV, the property mappings (date added, ip, do not email, etc) were ignored. This PR restores those but also adds them to the new format of showing the CSV fields to make the mapping a little more congruent. 

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Import the attached CSV and map the company, contact, date added and do not email fields
2. Note the errors
3. Import again without mapping the company fields and note that the do not email is not marked on the contact record


#### Steps to test this PR:
1. Repeat and no errors should occur and the do not contact flag should be set for scaddie0@theglobeandmail.com
[importcompany.csv.zip](https://github.com/mautic/mautic/files/1304699/importcompany.csv.zip)
